src/main/java/org/spongepowered/mod/mixin/world/MixinWorld.java

```diff
@@ -104,8 +104,7 @@ public BlockLoc getBlock(int x, int y, int z) {
         return new BlockWrapper(this, x, y, z);
     }
 
-    @Override
-    public Biome getBiome(Vector3d position) {
+    public BiomeType getBiome(Vector3d position) {
```

Comment:
Don't update the subproject then
cd into SpongeAPI and `git checkout 63e7cba4494a05ceb968c61b81f46d986c9101a3`